cmd/restic/global.go

```diff
@@ -111,8 +121,14 @@ func init() {
 	f.BoolVar(&globalOptions.CleanupCache, "cleanup-cache", false, "auto remove old cache directories")
 	f.IntVar(&globalOptions.LimitUploadKb, "limit-upload", 0, "limits uploads to a maximum rate in KiB/s. (default: unlimited)")
 	f.IntVar(&globalOptions.LimitDownloadKb, "limit-download", 0, "limits downloads to a maximum rate in KiB/s. (default: unlimited)")
+	f.UintVar(&globalOptions.MinPackSize, "min-packsize", 0, "set min pack size in MiB. (default: $RESTIC_MIN_PACKSIZE or 4)")
```

Comment:
should we set the default here to `4`?

Reply:
I suppose that would be good.

Reply:
So this flag defaults to zero because of line 128, where it uses the zero default to determine that flag has not been defined.  Normally, I would use -1 for this, but since globalOptions.MinPackSize is defined as a uint, we cannot use that.